pkg/apis/serving/v1/revision_lifecycle_test.go

```diff
@@ -575,6 +575,31 @@ func TestPropagateAutoscalerStatus(t *testing.T) {
 	apistest.CheckConditionSucceeded(r, RevisionConditionResourcesAvailable, t)
 }
 
+func TestPropagateAutoscalerStatusNoProgress(t *testing.T) {
+	r := &RevisionStatus{}
+	r.InitializeConditions()
+	apistest.CheckConditionOngoing(r, RevisionConditionReady, t)
+
+	// PodAutoscaler is not ready and initial scale was never attained.
+	r.PropagateAutoscalerStatus(&av1alpha1.PodAutoscalerStatus{
+		Status: duckv1.Status{
+			Conditions: duckv1.Conditions{{
+				Type:   av1alpha1.PodAutoscalerConditionReady,
+				Status: corev1.ConditionFalse,
+			}, {
+				Type:   av1alpha1.PodAutoscalerConditionScaleTargetInitialized,
+				Status: corev1.ConditionFalse,
```

Comment:
```suggestion
				Status: corev1.ConditionUnknown,
```

Not very important but the ScaleTargetInitialized condition starts off as unknown. I don't think we ever mark it as false.

Reply:
Yeah, it doesn't matter, but I can fix that.